src/pybind/mgr/dashboard/controllers/multi_cluster.py

```diff
@@ -266,6 +267,15 @@ def is_token_expired(self, jwt_token):
         current_time = time.time()
         return expiration_time < current_time
 
+    def get_time_left(self, jwt_token):
+        split_message = jwt_token.split(".")
+        base64_message = split_message[1]
+        decoded_token = json.loads(base64.urlsafe_b64decode(base64_message + "===="))
```

Comment:
@nizamial09 , i don't remember the exact reason but decoding token with this decode method was showing invalid token everytime maybe because of some pre-defined algos used, I didn't want to modify that part of the code, so went with this approach